modules/denoise/denoise_wrapper.cpp

```diff
@@ -0,0 +1,69 @@
+/*************************************************************************/
+/*  denoise_wrapper.cpp                                                  */
+/*************************************************************************/
+/*                       This file is part of:                           */
+/*                           GODOT ENGINE                                */
+/*                      https://godotengine.org                          */
+/*************************************************************************/
+/* Copyright (c) 2007-2020 Juan Linietsky, Ariel Manzur.                 */
+/* Copyright (c) 2014-2020 Godot Engine contributors (cf. AUTHORS.md).   */
+/*                                                                       */
+/* Permission is hereby granted, free of charge, to any person obtaining */
+/* a copy of this software and associated documentation files (the       */
+/* "Software"), to deal in the Software without restriction, including   */
+/* without limitation the rights to use, copy, modify, merge, publish,   */
+/* distribute, sublicense, and/or sell copies of the Software, and to    */
+/* permit persons to whom the Software is furnished to do so, subject to */
+/* the following conditions:                                             */
+/*                                                                       */
+/* The above copyright notice and this permission notice shall be        */
+/* included in all copies or substantial portions of the Software.       */
+/*                                                                       */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,       */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#include "denoise_wrapper.h"
+#include "core/os/copymem.h"
+#include "core/os/memory.h"
+#include "thirdparty/oidn/include/OpenImageDenoise/oidn.h"
+#include <stdio.h>
```

Comment:
```suggestion
#include "denoise_wrapper.h"

#include "core/os/copymem.h"
#include "core/os/memory.h"

#include "thirdparty/oidn/include/OpenImageDenoise/oidn.h"

#include <stdio.h>
```

See https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes

Note that normally you shouldn't include thirdparty headers with a fully qualified path like this, it would be better to add them to the include path and use `#include <OpenImageDenoise/oidn.h>`. But given that it's done the same way in `master`, no need to change it there. One day I'll get to allow unbundling this dependency on Linux distros and that will be part of it).